### PR TITLE
Add tooly composer script

### PR DESCRIPTION
**Pull request by tommy-muehle**
With the tooly composer-script aka hook you can manage needed PHAR files, such as phpunit, in your project composer.json. Every PHAR file will be saved in the composer binary directory. Optional with GPG verification.

### CODE REVIEW

**Comment by hollodotme** [comment]
+1

**Comment by mesutaslan** [comment]
+1

Mesut ASLAN

8 Eyl 2016 tarihinde 09:54 saatinde, Holger Woltersdorf <notifications@github.com> şunları yazdı:

**Comment by freshp** [comment]
+1

**Comment by josegonzalez** [comment]
In case you weren't aware, there's an emoji 👍 button that I highly recommend you all use.

File: README.md [hunk 1]
```diff
@@ -100,6 +100,7 @@ Please see [CONTRIBUTING](https://github.com/ziadoz/awesome-php/blob/master/CONT
 * [Prestissimo](https://github.com/hirak/prestissimo) - A composer plugin which enables parallel install process.
 * [Satis](https://github.com/composer/satis) - A static Composer repository generator.
 * [Toran Proxy](https://toranproxy.com) - A static Composer repository and proxy.
+* [tooly](https://github.com/tommy-muehle/tooly-composer-script) - A library to manage PHAR files in project using Composer.
```
Comment (josegonzalez) [inline README.md]: Needs to go in alphabetical order.

Comment (tommy-muehle) [inline README.md]: Fixed.

**Pull request merged**